packages/core/bootstrap/README.md

```diff
@@ -88,14 +88,17 @@ For example, if the `CACHE_KEY_IGNORED_PROPS=timestamp` is set, these requests w
 
 ### Redis
 
-| Required? |          Name          |                                                                   Description                                                                   | Options | Defaults to |
-| :-------: | :--------------------: | :---------------------------------------------------------------------------------------------------------------------------------------------: | :-----: | :---------: |
-|           |   `CACHE_REDIS_HOST`   |                                                         IP address of the Redis server.                                                         |         | `127.0.0.1` |
-|           |   `CACHE_REDIS_PORT`   |                                                            Port of the Redis server.                                                            |         |   `6379`    |
-|           |   `CACHE_REDIS_PATH`   |                                                   The UNIX socket string of the Redis server.                                                   |         |  undefined  |
-|           |   `CACHE_REDIS_URL`    | The URL of the Redis server. Format: `[redis[s]:]//[[user][:password@]][host][:port][/db-number][?db=db-number[&password=bar[&option=value]]]`. |         |  undefined  |
-|           | `CACHE_REDIS_PASSWORD` |                                                      The password required for redis auth.                                                      |         |   `null`    |
-|           | `CACHE_REDIS_TIMEOUT`  |                                      The timeout in ms if connection to Redis errors or is not responding.                                      |         |    `500`    |
+| Required? |                 Name                 |                                                                   Description                                                                   | Options | Defaults to |
+| :-------: | :----------------------------------: | :---------------------------------------------------------------------------------------------------------------------------------------------: | :-----: | :---------: |
+|           |   `CACHE_REDIS_CONNECTION_TIMEOUT`   |                                                     Timeout per long lived connection (ms)                                                      |         |   `15000`   |
```

Comment:
Will this make it reconnect every 15s, or how is this different from `CACHE_REDIS_TIMEOUT`?

Reply:
The `CACHE_REDIS_TIMEOUT` is a config on our side to determine when any given request is considered "timed-out". This `CACHE_REDIS_CONNECTION_TIMEOUT` is used as `connectTimeout` in the client to determine when to destroy the socket connection due to inactivity. [Here](https://github.com/redis/node-redis/blob/ac808ea781c4cdcc13e4e7a68777532d646b56c9/packages/client/lib/client/socket.ts#L154-L155) is where the client uses `connectTimeout`, and [here](https://nodejs.org/api/net.html#socketsettimeouttimeout-callback) is an explanation of `socket.setTimeout`.

Reply:
I can add more details in the README to make this clear

Reply:
70f5781

Reply:
Thanks for the explanation and readme clarification. Should be good now :+1: